expansion/src/main/resources/nviTypeQuery.sparql

```diff
@@ -13,11 +13,37 @@ CONSTRUCT {
     ?publication nva:status ?publicationStatus .
     ?publication nva:entityDescription/nva:reference/nva:publicationInstance/rdf:type ?instanceType .
 
-    OPTIONAL { ?publication nva:entityDescription/nva:publicationDate/nva:year ?publicationYear . }
-
-    OPTIONAL { ?publication nva:entityDescription/nva:reference/nva:publicationContext/nva:publisher/nva:level ?publisherLevel . }
-    OPTIONAL { ?publication nva:entityDescription/nva:reference/nva:publicationContext/nva:series/nva:level ?seriesLevel . }
-    OPTIONAL { ?publication nva:entityDescription/nva:reference/nva:publicationContext/nva:level ?journalLevel .  }
+    OPTIONAL {
```

Comment:
So it's said, we know that the limits of the graph don't include > 1 occurrence of these, so I guess these are fine. In truth, a check should be included to ensure we don't get weird results from e.g. artistic subtypes that include a publishing date (not sure if this is relevant any more…) I'm also pretty sure that these cases aren't NVI-data anyway…

Reply:
I agree, we can refactor these to make it more robust in the nva-nvi application